lib/rails2_ruby2/action_pack.rb

```diff
@@ -0,0 +1,3 @@
+Dir[File.dirname(__FILE__) + '/action_pack/**/*.rb'].each do |file|
+  require file.sub(/#{Regexp.escape(File.extname(file))}$/, '')
```

Comment:
given that we're globbing for `.rb`, i think it's safe to do `file.chomp('.rb')`

Reply:
Good call, much simpler. Updated in 7e66adaec09d99c3942d0f8006bd158c0e80a2f6